client.go

```diff
@@ -10,9 +10,19 @@ import (
 
 const Operations = "operations"
 const OperationsShort = "ops"
+const PartialFailureHeader = "x-partial-failure"
 
 type AppOpticsClient struct {
 	Token string
+	MeasurementsURI string
+}
+
+func NewAppOpticsClient(token, measurementsURI string) *AppOpticsClient {
+	if measurementsURI == "" {
+		measurementsURI = defaultMeasurementsURI
```

Comment:
this is a breaking API change as written; anyone creating the AppOpticsClient struct in the current release (where there's no `NewAppOpticsClient` method, and no `MeasurementsURI` field) will end up posting measurements to a URI of `""`. 

It may be kind of pedantic to be a semver snob but Go module resolution rules have kinda forced the issue for everyone :) 

if you do the defaulting at the time you're reading the MeasurementsURI instead of when you're creating the client (maybe with a `.getMeasurementsURI()` helper or something) then you can preserve the current behavior for existing clients that are constructing the struct as `&AppOpticsClient{token}` directly today. On the other hand, I don't think there are that many users of this in the wild so a breaking change is probably not a big deal...